package.json

```diff
@@ -83,6 +86,9 @@
     "warning": "^3.0.0"
   },
   "peerDependencies": {
+    "@gandi/babel-preset-gettext": "*",
+    "babel-cli": "*",
```

Comment:
@iamdey : I don't understand the debate here. If you agree it's a real dep, it should be removed from the peerDeps, no?